source/apiVolontaria/apiVolontaria/tests/tests_view_ResetPassword.py

```diff
@@ -272,37 +271,31 @@ def test_create_new_token_without_email_service(self, imailing):
             }
         }
     )
-    @mock.patch('apiVolontaria.views.IMailing')
-    def test_create_new_token_with_failure_on_email_service(self, imailing):
+
+    def test_create_new_token_with_failure_on_email_service(self):
         """
         Ensure we can have a new token to change our password
         """
         data = {
             'username': self.user.username,
         }
 
-        instance_imailing = imailing.create_instance.return_value
-        instance_imailing.send_templated_email.return_value = {
-            "code": "failure",
-        }
-
         response = self.client.post(
             reverse('reset_password'),
             data,
             format='json',
         )
+        # Test that one message was sent:
+        self.assertEqual(len(mail.outbox), 1)
+        self.assertEqual(mail.outbox[0].to, [self.user.email])
 
         # The token has been created
         tokens = ActionToken.objects.filter(
             user=self.user,
             type='password_change',
         )
 
-        content = {
-            'detail': "Your token has been created but no email "
-                      "has been sent. Please contact the administration.",
-        }
-        self.assertEqual(json.loads(response.content), content)
+        self.assertEqual(response.content, b'')
```

Comment:
We want a fail! Here you created a success!
Where is the old error message for the final user ?